router/router.go

```diff
@@ -586,8 +586,8 @@ func (self *Router) initializeHealthChecks() (gosundheit.Health, error) {
 	err = h.RegisterCheck(&linkHealthCheck{router: self, minLinks: checkConfig.LinkCheck.MinLinks},
 		gosundheit.ExecutionPeriod(checkConfig.LinkCheck.Interval),
 		gosundheit.ExecutionTimeout(5*time.Second),
-		gosundheit.InitiallyPassing(true),
-		gosundheit.InitialDelay(1*time.Second),
+		gosundheit.InitiallyPassing(checkConfig.LinkCheck.MinLinks == 0),
```

Comment:
a thought... feels like logic that should be in a separate function that's eventually testable/assert-able?

Reply:
Yes. I would wait to refactor it until we add the test, other wise it'll likely need to be refactored twice :)